frontend/packages/pipelines-plugin/src/utils/pipeline-filter-reducer.ts

```diff
@@ -2,22 +2,39 @@ import * as _ from 'lodash';
 
 export const pipelineRunStatus = (pipelineRun): string => {
```

Comment:
Can you add an documentation link here? Maybe like this:

```suggestion
// Converts the PipelineRun (and TaskRun) condition status into a human readable string.
// See also tkn cli implementation at https://github.com/tektoncd/cli/blob/release-v0.15.0/pkg/formatted/k8s.go#L54-L83
export const pipelineRunStatus = (pipelineRun): string => {
```

Reply:
Probably could have used `/** ... */` for an official JSDoc of the method... but that's fine.